crates/re_space_view_spatial/src/view_3d_properties.rs

```diff
@@ -0,0 +1,27 @@
+use re_types::{
+    blueprint::{archetypes::Background, components::BackgroundKind},
+    components::Color,
+    Archetype,
+};
+use re_viewer_context::TypedComponentFallbackProvider;
+
+use crate::SpatialSpaceView3D;
+
+impl TypedComponentFallbackProvider<Color> for SpatialSpaceView3D {
+    fn fallback_for(&self, ctx: &re_viewer_context::QueryContext<'_>) -> Color {
+        // Color is a fairly common component, make sure this is the right context.
+        if ctx.archetype_name == Some(Background::name()) {
+            Color::WHITE
```

Comment:
Is there a reason our 3D views default to white but our 2D views default to black?